cmake/third-party/easylogging/FindEASYLOGGINGPP.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+find_library(EASYLOGGINGPP_LIBRARY easyloggingpp)
+find_path(EASYLOGGINGPP_INCLUDEDIR easylogging++.h PATH_SUFFIXES include)
+
+find_package_handle_standard_args(EASYLOGGINGPP REQUIRED_VARS EASYLOGGINGPP_INCLUDEDIR)
+find_package_handle_standard_args(EASYLOGGINGPP_LIB REQUIRED_VARS EASYLOGGINGPP_LIBRARY)
+
+if(EASYLOGGINGPP_FOUND)
+    message(STATUS "easyloggingpp include directory found")
+else()
+    message(STATUS "easyloggingpp include directory not found, using EASYLOGGINGPP packaged with HEXL")
+endif()
+
+if(EASYLOGGINGPP_LIB_FOUND)
+    message(STATUS "easyloggingpp library found")
+else()
+    message(STATUS "easyloggingpp library not found, using EASYLOGGINGPP packaged with HEXL")
+endif()
+
+add_library(easyloggingpp::easyloggingpp INTERFACE IMPORTED)
```

Comment:
If you switch to `easyloggingpp` instead of `easyloggingpp::easyloggingpp` here and below, I'd think you could avoid the `add_library(easyloggingpp ALIAS easyloggingpp::easyloggingpp)` in the CMakeLists.txt file.